pkg/oci/utils.go

```diff
@@ -140,10 +143,34 @@ func networkConfig(ocispec spec.Spec) (vc.NetworkConfig, error) {
 
 // PodConfig converts an OCI compatible runtime configuration file
 // to a virtcontainers pod configuration structure.
-func PodConfig(runtime RuntimeConfig, bundlePath, cid, console string) (*vc.PodConfig, *spec.Spec, error) {
+func PodConfig(runtime RuntimeConfig, bundlePath, cid, console string, podAnnotations, containerAnnotations map[string]string) (*vc.PodConfig, *spec.Spec, error) {
```

Comment:
So here you want the caller (e.g. our 3.0 runtime implementation) to be responsible for creating both the pod and container annotations ?

I may be missing something, but the idea of this routine is to transform OCI specific data into virtcontainers pointers. So I wonder what is the rationale behind adding pod and container annotations as arguments here. Since this is being done in the `cc-runtime list` implementation context, there may be some annotations that you'd want to pass from the runtime through a pod or container config. In that case, this should be explicitly done from the runtime _after_ calling this routine by setting `Annotations` directly (or through some setter routine).